features/doctrine/date_filter.feature

```diff
@@ -29,13 +29,16 @@ Feature: Date filter on collections
                   {"pattern": "^/dummies/29$"}
                 ]
               }
-            },
-            "hydra:view": {
-              "@id": {"pattern": "^/dummies\\?dummyDate\\[after\\]=2015-04-28$"},
-              "@type": {"pattern": "^hydra:PartialCollectionView$"}
             }
           },
           "maxItems": 2
+        },
+        "hydra:view": {
```

Comment:
Yeah, the JSON Schema was incorrect/incomplete and did not actually check the `hydra:view` element. We should check the other features for similar issues.